client/profile-wizard/steps/theme/index.js

```diff
@@ -92,7 +92,10 @@ class Theme extends Component {
 			location,
 		} );
 
-		if ( slug !== activeTheme && getPriceValue( price ) <= 0 ) {
+		if (
+			slug !== activeTheme &&
+			( getPriceValue( price ) <= 0 || isInstalled )
+		) {
 			if ( isInstalled ) {
 				this.activateTheme( slug );
 			} else {
```

Comment:
Nitpick: I think that we could simplify this a  little bit by taking the `if ( isInstalled ) {`  out of the other `if`. It's frankly minimal, so I'm totally ok if you want to keep it like now.
The code would look like this:
```
	    if ( isInstalled ) {
			this.activateTheme( slug );
		} else {
			if ( slug !== activeTheme && getPriceValue( price ) <= 0 ) {
				this.installTheme( slug );
			} else {
				updateProfileItems( { theme: slug } );
			}
		}
```